components/src/atoms/ListButton/index.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react'
+import { css } from 'styled-components'
+import { Flex } from '../../primitives'
+import { SPACING } from '../../ui-style-constants'
+import { BORDERS, COLORS } from '../../helix-design-system'
+import type { StyleProps } from '../../primitives'
+
+export * from './ListButtonChildren/index'
+
+export type ListButtonType = 'noActive' | 'connected' | 'notConnected'
+
+interface ListButtonProps extends StyleProps {
+  type: ListButtonType
+  children: React.ReactNode
+  disabled?: boolean
+  onClick?: () => void
+}
+
+const LISTBUTTON_PROPS_BY_TYPE: Record<
+  ListButtonType,
+  { backgroundColor: string; hoverBackgroundColor: string }
+> = {
+  noActive: {
+    backgroundColor: COLORS.grey35,
+    hoverBackgroundColor: COLORS.grey40,
+  },
+  connected: {
+    backgroundColor: COLORS.green35,
+    hoverBackgroundColor: COLORS.green35,
+  },
+  notConnected: {
+    backgroundColor: COLORS.yellow35,
+    hoverBackgroundColor: COLORS.yellow40,
+  },
```

Comment:
https://www.figma.com/design/WbkiUyU8VhtKz0JSuIFA45/Feature%3A-Protocol-Designer-Phase-1?node-id=6221-148509&m=dev
```suggestion
  noActive: {
    backgroundColor: COLORS.grey30,
    hoverBackgroundColor: COLORS.grey35,
  },
  connected: {
    backgroundColor: COLORS.green30,
    hoverBackgroundColor: COLORS.green35,
  },
  notConnected: {
    backgroundColor: COLORS.yellow30,
    hoverBackgroundColor: COLORS.yellow35,
  },
```